CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 
 ### New features
 
+* [#5444](https://github.com/bbatsov/rubocop/pull/5444): Add new `Style/AccessModifierDeclarations` cop. ([@brandonweiss][])
```

Comment:
One small note - this the changelog for the previous RuboCop release. :-)

Reply:
Whoops! Fixing.